app/controllers/orders_controller.rb

```diff
@@ -15,6 +15,9 @@ def create
         stripe_token: params[:stripeToken]
       )
     )
+    if current_user.credit_card.card_token.nil?
+      @order.save_card
+    end
```

Comment:
This is up to you, and I don't know if there's an SB standard, but when there's only one action being called from a conditional, we usually write those one line. For example:
`@order.save_card if current_user.credit_card.card_token.nil?`